inventory/os.cf

```diff
@@ -14,4 +19,20 @@ _inventory_lsb_found::
   "description" string => "$(sys.flavor) (LSB missing)",
                   meta => { "inventory", "attribute_name=OS" };
 
+# Hard coded values for exceptions / platforms without os-release:
+
+centos_5::
+  "description" string => "CentOS Linux 5", # Matches format of os-release on 7+
+                  meta => { "inventory", "attribute_name=OS,derived-from=centos_5" };
+
+centos_6::
```

Comment:
This will need to be deconflated for redat vs centos

Reply:
I didn't include redhat on purpose, since I don't have redhat to test on. Only implementing what I've tested.